### DISCUSSION TITLE
Thanks for Lox365.  Are you interested in another small function?

### DISCUSSION THREAD

**Discussion by bruce809** [goosepirate/lox365#3] [General]
Hi Goose_pirate,
  
Thank you for developing Lox365.  You have more ability in this area than I do.  If you would you consider helping code (5-10 minutes) or coding a small function for LibreOffice, please see   
https://github.com/bruce809/Repository4bruce.
  
A reference is https://ask.libreoffice.org/t/could-typein-compliment-the-formula-function-typein-was-transfer-in-the-original-question/82370
  
Cordially,
Bruce

**Comment by goosepirate** [comment]
Hi Bruce, if I understand you correctly, you want to make a new Calc function that evaluates a given formula string and then modifies another cell based on the results. Is that right?

**Comment by bruce809** [comment]
Hi goosepirate,  
Thank you for responding.  You are exactly right. 
  
Bruce
   

            
                   
                  
                 
                 
               
                
               
           
             


    On Thursday, October 6, 2022 at 01:00:37 AM EDT, goosepirate ***@***.***> wrote:  
 
 


Hi Bruce, if I understand you correctly, you want to make a new Calc function that evaluates a given formula string and then modifies another cell based on the results. Is that right?

**Comment by goosepirate** [comment]
Hi Bruce, I think this is technically possible to do but unfortunately at the moment I'm not familiar enough with LibreOffice to be able to code this, sorry. Anyway, thank you for the interest.

**Reply by bruce809** [reply]
goosepirate, Thanks for looking at this.  You are right.  The basic question is "Is this feasible?"  Thank you for answering that question.  The secondary question is "Would you like to code it?"  Thank you for considering this question as well.  As far as I am concerned, you have been both kind and quite gracious.  Thank you for the both the help, and the encouragement and the additional knowledge you have given to promote this project.
Cordially, Bruce
